WORKING-GROUP-PROCESSES.md

```diff
@@ -0,0 +1,143 @@
+# Istio Working Group Processes
+
+This document describes the processes we use to manage the Istio working groups. This includes how they are formed, how leads
+are established, how they are run, etc.
+
+* [Why working groups?](#why-working-groups)
+* [Proposing a new working group](#proposing-a-new-working-group)
+* [Setting up a working group](#setting-up-a-working-group)
+  * [Dissolving a working group](#dissolving-a-working-group)
+* [Running a working group](#running-a-working-group)
+  * [Be open](#be-open)
+  * [Making decisions](#making-decisions)
+  * [Subgroups](#subgroups)
+  * [Escalations](#escalations)
+
+## Why working groups?
+
+Istio working groups are organizations responsible for the design and implementation of large architectural aspects of the overall Istio project.
+Working groups operate with a fair amount of autonomy within the broader scope of the project. They tend to be long-lived, representing major
+initiatives over Istio’s lifetime.
+
+Some working groups focus on specific technologies. For example, the Integrations working group primarily focuses on Mixer. Other working
+groups are cross-cutting in nature such as the Performance and Scalability working group.
+
+The Steering working group is responsible for the Istio project as a whole. It sets the overall direction of the project,
```

Comment:
Random note, I've been wondering if we should call the steering group the steering committee instead of steering working group?  It isn't really a working group in the normal sense. We should discuss in the steering committee meeting maybe, but we don't want to accidentally make that group subject to all the same rules.

K8s has a similar model, the steering committee is not a SIG, it is a separate thing with a different model (a committee).